fixtures/ibiza-light/pluton10-lzr.json

```diff
@@ -0,0 +1,497 @@
+          "slotNumber": 4,
+          "comment": "Gobo 4"
+        },
+        {
+          "dmxRange": [32, 39],
+          "type": "WheelSlot",
+          "slotNumber": 5,
+          "comment": "Gobo 5"
+        },
+        {
+          "dmxRange": [40, 47],
+          "type": "WheelSlot",
+          "slotNumber": 6,
+          "comment": "Gobo 6"
+        },
+        {
+          "dmxRange": [48, 55],
+          "type": "WheelSlot",
+          "slotNumber": 7,
+          "comment": "Gobo 7"
+        },
+        {
+          "dmxRange": [56, 63],
+          "type": "WheelSlot",
+          "slotNumber": 8,
+          "comment": "Gobo 8"
+        },
+        {
+          "dmxRange": [64, 71],
+          "type": "WheelShake",
+          "slotNumber": 9,
+          "comment": "Gobo 1 shake"
+        },
+        {
+          "dmxRange": [72, 79],
+          "type": "WheelShake",
+          "slotNumber": 10,
+          "comment": "Gobo 2 shake"
+        },
+        {
+          "dmxRange": [80, 87],
+          "type": "WheelShake",
+          "slotNumber": 11,
+          "comment": "Gobo 3 shake"
+        },
+        {
+          "dmxRange": [88, 95],
+          "type": "WheelShake",
+          "slotNumber": 12,
+          "comment": "Gobo 4 shake"
+        },
+        {
+          "dmxRange": [96, 103],
+          "type": "WheelShake",
+          "slotNumber": 13,
+          "comment": "Gobo 5 shake"
+        },
+        {
+          "dmxRange": [104, 111],
+          "type": "WheelShake",
+          "slotNumber": 14,
+          "comment": "Gobo 6 shake"
+        },
+        {
+          "dmxRange": [112, 119],
+          "type": "WheelShake",
+          "slotNumber": 15,
+          "comment": "Gobo 7 shake"
+        },
+        {
+          "dmxRange": [120, 127],
+          "type": "WheelShake",
+          "slotNumber": 16,
+          "comment": "Gobo 8 shake"
+        },
+        {
+          "dmxRange": [128, 189],
+          "type": "WheelSlotRotation",
+          "speedStart": "fast",
+          "speedEnd": "slow",
+          "comment": "Gobo Scroll"
+        },
+        {
+          "dmxRange": [190, 193],
+          "type": "WheelSlotRotation",
+          "speed": "stop"
+        },
+        {
+          "dmxRange": [194, 255],
+          "type": "WheelSlotRotation",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "Gobo Scroll "
+        }
+      ]
+    },
+    "Laser Motor (4 gobos)": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "WheelSlotRotation",
+          "speed": "stop"
+        },
+        {
+          "dmxRange": [8, 131],
+          "type": "WheelSlot",
+          "slotNumber": 2,
+          "comment": "Forward rotate"
+        },
+        {
+          "dmxRange": [132, 255],
+          "type": "WheelSlot",
+          "slotNumber": 3,
+          "comment": "Reverse rotate"
+        }
+      ]
+    },
+    "Strobe": {
```

Comment:
```
"Strobe": {
  "capabilities": [
    {
      "dmxRange": [0, 7],
      "type": "Closed"
    },
    {
      "dmxRange": [8, 15],
      "type": "Open"
    },
    {
      "dmxRange": [16, 131],
      "type": "Strobe",
      "speedStart": "slow",
      "speedEnd": "fast"
    },
    {
      "dmxRange": [132, 139],
      "type": "Open"
    },
    {
      "dmxRange": [140, 181],
      "type": "RampUp",
      "speedStart": "slow",
      "speedEnd": "fast"
    },
    {
      "dmxRange": [182, 189],
      "type": "Open"
    },
    {
      "dmxRange": [190, 231],
      "type": "RampDown",
      "speedStart": "slow",
      "speedEnd": "fast"
    },
    {
      "dmxRange": [232, 239],
      "type": "Open"
    },
    {
      "dmxRange": [240, 247],
      "type": "Strobe",
      "speedStart": "slow",
      "speedEnd": "fast",
      "randomTiming": true
    },
    {
      "dmxRange": [248, 255],
      "type": "Open"
    },
  ]
}
```

I'm guessing that all the ranges are slow...fast, but only the strobe (16-131) actually says that in the manual.